Formula/chart-testing.rb

```diff
@@ -0,0 +1,35 @@
+class ChartTesting < Formula
+  desc "Testing and linting Helm charts"
+  homepage "https://github.com/helm/chart-testing"
+  url "https://github.com/helm/chart-testing.git",
+      {
+        tag:      "v3.0.0",
+        revision: "50db473a1e68c605b18d82f019d83ea401542213",
+      }
```

Comment:
@Homebrew/core is this correct with the new hash syntax?

Reply:
@SeekingMeaning Yes it does but I changed it according to @SeekingMeaning's idea.